src/cwv/handler.js

```diff
@@ -44,7 +45,81 @@ export async function CWVRunner(auditUrl, context, site) {
   };
 }
 
+export async function convertToOppty(auditUrl, auditData, context) {
+  const {
+    dataAccess,
+    log,
+  } = context;
+
+  log.info(`auditUrl: ${auditUrl}`);
+  log.info(`auditData: ${JSON.stringify(auditData)}`);
+
+  const opportunities = await dataAccess.Opportunity.allBySiteIdAndStatus(auditData.siteId, 'NEW');
+  log.info(`opportunities: ${JSON.stringify(opportunities)}`);
+
+  let opportunity = opportunities.find((oppty) => oppty.getType() === AUDIT_TYPE);
+  log.info(`opportunity: ${JSON.stringify(opportunity)}`);
+
+  if (!opportunity) {
+    const opportunityData = {
+      siteId: auditData.siteId,
+      auditId: auditData.id,
+      runbook: 'https://adobe.sharepoint.com/sites/aemsites-engineering/Shared%20Documents/3%20-%20Experience%20Success/SpaceCat/Runbooks/Experience_Success_Studio_CWV_CLS_Runbook.docx?web=1',
```

Comment:
Do we need to move the runbook URLs to a dedicated config file?

Reply:
if the core team decides to arrange runbooks in a such fashion, we can adapt this line later on. cc/ @solaris007, @alinarublea